griptape/drivers/prompt_model/bedrock_claude_prompt_model_driver.py

```diff
@@ -0,0 +1,49 @@
+import json
+from attr import define, field, Factory
+from griptape.artifacts import TextArtifact
+from griptape.utils import PromptStack
+from griptape.drivers import BasePromptModelDriver
+from griptape.tokenizers import AnthropicTokenizer
+
+
+@define
+class BedrockClaudePromptModelDriver(BasePromptModelDriver):
+    tokenizer: AnthropicTokenizer = field(
+        default=Factory(
+            lambda: AnthropicTokenizer(model="anthropic.claude-v2"),
```

Comment:
Should we move model into a class property since Bedrock supports claude-v1 as well? And will likely support claude-v3 in the future.